src/test/projects/databinding/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                              http://maven.apache.org/maven-v4_0_0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <groupId>com.simpligility.android.databinding</groupId>
+   <artifactId>parent</artifactId>
+   <version>1.0.0-SNAPSHOT</version>
+   <packaging>pom</packaging>
+
+   <name>Data Binding Integration test :: Parent</name>
+
+   <modules>
+      <module>library</module>
+      <module>app</module>
+   </modules>
+
+   <properties>
+      <it-plugin.version>4.3.1-SNAPSHOT</it-plugin.version>
+   </properties>
+
+   <dependencyManagement>
+      <dependencies>
+         <dependency>
+            <groupId>android</groupId>
+            <artifactId>android</artifactId>
+            <version>6.0_r1</version>
+            <scope>provided</scope>
+         </dependency>
+         <dependency>
+            <groupId>com.android.databinding</groupId>
+            <artifactId>adapters</artifactId>
+            <version>1.0-rc2</version>
+            <type>aar</type>
+            <exclusions>
+               <exclusion>
+                  <groupId>com.android.databinding</groupId>
+                  <artifactId>library</artifactId>
+               </exclusion>
+            </exclusions>
+         </dependency>
+         <dependency>
+            <groupId>com.android.databinding</groupId>
+            <artifactId>library</artifactId>
+            <version>1.0-rc2</version>
+            <type>aar</type>
+            <exclusions>
+               <exclusion>
+                  <groupId>com.android.support</groupId>
+                  <artifactId>support-v4</artifactId>
+               </exclusion>
+            </exclusions>
+         </dependency>
+         <dependency>
+            <groupId>com.android.databinding</groupId>
+            <artifactId>compiler</artifactId>
+            <version>1.0-rc2</version>
+            <type>jar</type>
+            <scope>provided</scope>
+         </dependency>
+         <dependency>
+            <groupId>com.android.support</groupId>
+            <artifactId>support-v4</artifactId>
+            <version>23.0.1</version>
+            <type>aar</type>
+         </dependency>
+         <dependency>
+            <groupId>com.android.support</groupId>
+            <artifactId>appcompat-v7</artifactId>
+            <version>23.0.1</version>
+            <type>aar</type>
+         </dependency>
+         <dependency>
+            <groupId>com.android.support</groupId>
+            <artifactId>design</artifactId>
+            <version>23.0.1</version>
+            <type>aar</type>
+         </dependency>
+         <dependency>
+            <groupId>com.android.support</groupId>
+            <artifactId>support-annotations</artifactId>
+            <version>23.0.1</version>
```

Comment:
great. the biggest issue is that the annotation processor looks through the
filesystem for some files, not specified folders.  result is that build
always needs to be run from the root of module, I suppose Gradle might
guarantee that.. well maven doesn't.  I'm not even sure how to fix it and I
haven't found the code for the actual annotation processor.  I have aosp
source pulled I need to just search for it there..
On Jan 26, 2016 12:26 AM, "Manfred Moser" <notifications@github.com> wrote:

> In src/test/projects/databinding/pom.xml
> <https://github.com/simpligility/android-maven-plugin/pull/685#discussion_r50799025>
> :
>
> > +         <dependency>
> > +            <groupId>com.android.support</groupId>
> > +            <artifactId>appcompat-v7</artifactId>
> > +            <version>23.0.1</version>
> > +            <type>aar</type>
> > +         </dependency>
> > +         <dependency>
> > +            <groupId>com.android.support</groupId>
> > +            <artifactId>design</artifactId>
> > +            <version>23.0.1</version>
> > +            <type>aar</type>
> > +         </dependency>
> > +         <dependency>
> > +            <groupId>com.android.support</groupId>
> > +            <artifactId>support-annotations</artifactId>
> > +            <version>23.0.1</version>
>
> Should be using latest version..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simpligility/android-maven-plugin/pull/685/files#r50799025>
> .
>